src/lib/constants.ts

```diff
@@ -33,3 +33,57 @@ export const USER_TYPE_TO_DISPLAY_NAME: Record<AcsUserExternalType, string> = {
   hid_credential_manager_user: "HID user",
   salto_site_user: "Salto site user",
 }
+
+export const SEAM_EVENT_LIST = [
```

Comment:
Not specific to this PR but how do we plan on keeping this list (and things like it) up to date?

Reply:
You're right, we currently lack a mechanism for updating the `SEAM_EVENT_LIST` and similar things on fake-seam-connect. We should discuss and establish a process to ensure these lists stay up-to-date with the production seam-connect codebase. Looking forward to ideas and suggestions.